Rx/v2/src/rxcpp/operators/impl-retry-repeat.hpp

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+/*! \file impl-retry-repeat.hpp
+
+    \brief Implementation commonalities between retry and repeat operators abstracted away from rx-retry.hpp and rx-repeat.hpp files. Should be used only from rx-retry.hpp and rx-repeat.hpp
+    
+*/
+
+#include "../rx-includes.hpp"
+
+namespace rxcpp {
+  namespace operators {
+    namespace detail {
+
+      namespace retry_repeat_common {
```

Comment:
I would prefer for the name of the file to match the name of the type or namespace that is introduces. This preference here does not block merge of this PR.